BUILDING.md

```diff
@@ -26,7 +26,7 @@ Support is divided into three tiers:
   the broader community.
 * **Tier 2**: Full test coverage but more limited maintenance,
   often provided by the vendor of the platform.
-* **Experimental**: Known to compile but not necessarily reliably or with
+* **Experimental**: Known to compile but not necessarily reliable or with
```

Comment:
I think `reliably` is the correct word here. A fuller sentence would be: This is known to compile but it does not necessarily compile reliably or with a full passing test suite.

That said, all the qualifiers (`known to`, `not necessarily`, `reliably`...) are redundant. Maybe replace the whole sentence with something like this?:

> May not compile reliably. Test suite may not pass.

Concise and clear.